src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -41,24 +41,24 @@ export interface IGridDataSet {
 // tslint:disable:max-line-length
 const template = `
 <div class="fullsize vertBox">
-    <div *ngIf="dataSets.length > 0" role="button" id="resultspane" tabIndex="0" class="boxRow header collapsible"
+    <div *ngIf="dataSets.length > 0" role="button" id="resultspane" tabIndex="1" class="boxRow header collapsible"
```

Comment:
we don't usually do this. tab index=0 will make the element keyboard focusable, and the order will be determined by the its position in the DOM tree. please keep this as 0.